test/integration/tox.ini

```diff
@@ -3,13 +3,11 @@ envlist = integration-tests, flake8
 skipsdist = True
 
 [testenv]
-recreate=True
 skip_install = True
-passenv = DISPLAY MOZ_HEADLESS
 deps = -rpipenv.txt
 commands =
     pipenv install --skip-lock
-    pipenv run pytest -v --verify-base-url -n 2 --driver Firefox --html=send-test.html --self-contained-html {posargs}
+    pipenv run pytest -v --verify-base-url --driver Remote --capability browserName firefox --host selenium --html=send-test.html --self-contained-html {posargs}
```

Comment:
The driver is remote but isn't firefox within the same image?

Reply:
Selenium is running in it's own (separate) container so that's why this is remote. You could instead base the integration-tests image on the selenium one but that'd make the new Dockerfile more complicated because it'd have to run multiple processes. I figured this way was a bit simpler.